.github/workflows/update_permissions_table.yaml

```diff
@@ -0,0 +1,39 @@
+name: Update Permissions Tables
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+jobs:
+  refersh_books:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Update Files
+      id: update
+      env:
+        WEB3_INFURA_PROJECT_ID = ${{ secrets.INFURA_KEY }}
```

Comment:
@mikemcdonald this secret needs to be setup with an infura project key (free account is fine).  You can also clearly rename the secret here if there already is one.

Otherwise should run.  This one takes a while to poll everything.